MailCore/Utils/Constants.swift

```diff
@@ -161,8 +161,16 @@ public enum Constants {
         let release = Bundle.main.object(forInfoDictionaryKey: "CFBundleShortVersionString") as! String? ?? "x.x"
         let build = Bundle.main.object(forInfoDictionaryKey: "CFBundleVersion") as! String? ?? "x"
         let betaRelease = Bundle.main.isRunningInTestFlight ? "beta" : ""
+        let platformDetector = PlatformDetector()
```

Comment:
You can take a look at other files where we use PlatformDetector. 
We do not create an instance directly, instead, we use the dependency injection like this
`@InjectService private var platformDetector: PlatformDetectable`